vro-streamlit/src/vro_streamlit/auth/auth_service.py

```diff
@@ -1,9 +1,115 @@
+import logging
+import time
+from typing import Any
+
+import requests
+
+from vro_streamlit.auth.auth_exception import ForbiddenException, UnauthorizedException
+from vro_streamlit.auth.response_models import DeviceFlowResponse, UserInfoResponse
 from vro_streamlit.auth.user import User
+from vro_streamlit.config import GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET
+
+DEVICE_CODE_URL = 'https://github.com/login/device/code'
+TOKEN_URL = 'https://github.com/login/oauth/access_token'
+USER_URL = 'https://api.github.com/user'
+USER_ORG_URL = 'https://api.github.com/user/orgs'
+ORG = 'department-of-veterans-affairs'
```

Comment:
`ORG = 'department-of-veterans-affairs'` is also defined in `config.py`. We could import it instead:
`from vro_streamlit.config import ORG`

Reply:
Ah. I meant to remove it from `config.py` since its really never going to change ==> meaning its a CONSTANT!